src/main/java/com/appman/nahug/model/CatModel.java

```diff
@@ -94,4 +92,27 @@ public void setGender(String gender) {
         this.gender = gender;
     }
 
+    public UserModel getUserModelId() {
+        return userModelId;
+    }
+
+    public void setUserModelId(UserModel userModelId) {
```

Comment:
parameter name should be more define than this
I think `userModelId` should be like `userMode`